cmake/msys/msys_helper.sh

```diff
@@ -222,9 +212,8 @@ if [ ! -e "$mingw_root/lib/libgig/libgig.dll.a" ]; then
 	make install
 
 	mv "$mingw_root/lib/bin/libakai-0.dll" "$mingw_root/bin"
-	mv "$mingw_root/lib/bin/libgig-7.dll" "$mingw_root/bin"
 
-	if [ $? -ne 0 ]; then
+	if ! mv "$mingw_root/lib/bin/libgig-7.dll" "$mingw_root/bin"; then
```

Comment:
@zapashcanon I think so.
If you add `set -e`, however, printing custom error message will be unable. Is there alternative method? Or, is it okay to disable printing custom error message?